docs/architecture.md

```diff
@@ -210,7 +210,7 @@ this function. Most notably:
 
 - [Cosmos Relayer](https://github.com/cosmos/relayer):
   A fully functional relayer written in Go.
-- [Babylon Relayer](https://github.com/babylonlabs-io/babylon-relayer/):
+- [Babylon Relayer](https://github.com/babylonlabs-io/babylon/blob/f23219536b996b308a59923a4e5716f2a090303d/docs/ibc-relayer.md):
```

Comment:
why not use main https://github.com/babylonlabs-io/babylon/blob/main/docs/ibc-relayer.md ?